pkg/pod-autoscaler/pkg/recommender/controller.go

```diff
@@ -130,7 +130,7 @@ func (c *Controller) Run(threadiness int, stopCh <-chan struct{}) error {
 	for i := 0; i < threadiness; i++ {
 		go wait.Until(c.runNodeRecommenderWorker, time.Second, stopCh)
 	}
-	go wait.Until(c.runRecommenderWorker, 5*time.Second, stopCh)
+	go wait.Until(c.runRecommenderWorker, 6*time.Second, stopCh)
```

Comment:
Do we want to add a few seconds more? Just for safety reasons. Or are 6 seconds more than enough to handle pod changes

Reply:
6 is a good timing (from previous benchmark runs)